charts/security-apps/values.yaml

```diff
@@ -18,8 +18,30 @@ dex:
   # @default -- [upstream values](https://github.com/dexidp/helm-charts/tree/master/charts/dex/values.yaml)
   values: {}
 
-# gangway -- [gangway](https://github.com/heptiolabs/gangway/) ([example](./examples/gangway.yaml))
+# dexK8sAuthenticator -- [dex-k8s-authenticator](https://github.com/mintel/dex-k8s-authenticator) ([example](./examples/dex-k8s-authenticator.yaml))
 # @default -- -
+dexK8sAuthenticator:
+  # dexK8sAuthenticator.enabled -- Enable dex-k8s-authenticator
+  enabled: false
+  name: dex-k8s-authenticator
+  destination:
+    # dexK8sAuthenticator.destination.namespace -- Namespace
+    namespace: "infra-dex-k8s-authenticator"
+  # dexK8sAuthenticator.repoURL -- Repo URL
+  # @default -- [repo](https://github.com/mintel/dex-k8s-authenticator.git)
+  repoURL: "https://github.com/mintel/dex-k8s-authenticator.git"
```

Comment:
i found a valid repo in https://artifacthub.io/packages/helm/sagikazarmark/dex-k8s-authenticator which would be `https://charts.sagikazarmark.dev`

Reply:
Yeah, but that is pointing to a user's repo.. dont think we should use any user's repo anywhere, this could impact the infrastucture in case the user wants to delete the image or even create security issues. I think we should stick to the main repo or open some issues and provide the solution for this, not sure what the others think about it..

Reply:
that makes sense! thanks
but could be the same with the github repo right? :)

Reply:
Yep, but at least is the offical repo, so we somehow get more guarantee, but again, this is only my opinion, doesn't mean this is the way to go :)

Reply:
right, looks good to me then :)

Reply:
imo this needs a comment that  explains the rationale behind it being the 1 (one) -apps chart that uses a github directly and thus can't wildcard semver versions.

the comment has to explain that there is no repo for the thing and it must also include some links to upstream issues discussing this.

Reply:
let me know if you think I should add more coments in there

Reply:
thanks for reaching out to upstream and adding comments!